xgboost-predictor/src/main/java/biz/k11i/xgboost/Predictor.java

```diff
@@ -153,17 +178,17 @@ void initObjGbm() {
      * @return prediction values
      */
     public float[] predict(FVec feat, boolean output_margin, int ntree_limit) {
-        float[] preds = predictRaw(feat, ntree_limit);
+        float[] preds = predictRaw(feat, mparam.base_score, ntree_limit);
         if (! output_margin) {
             preds = obj.predTransform(preds);
         }
         return preds;
     }
 
-    float[] predictRaw(FVec feat, int ntree_limit) {
+    float[] predictRaw(FVec feat, float base_score, int ntree_limit) {
         float[] preds = gbm.predict(feat, ntree_limit);
         for (int i = 0; i < preds.length; i++) {
-            preds[i] += mparam.base_score;
```

Comment:
this seems incorrect to me, is there any other place where the base score would be applied (base_score IMHO corresponds to H2O's GBM `init_f`)?